src/index.html

```diff
@@ -1,12 +1,420 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css">
+    <link rel="stylesheet" href="./styles/blocks/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__inner">
+        <a href="#" class="header__logo">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+            class="header__logo-image"
+          >
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="nav__link nav__link--is-active is-active">
+                Apple
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">
+                Samsung
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#smartphones" class="nav__link">
+                Smartphones
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#laptops"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                Laptops & Computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#gadgets" class="nav__link">
+                Gadgets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#tablets" class="nav__link">
+                Tablets
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#photo" class="nav__link">
+                Photo
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#video" class="nav__link">
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="catalog">
+      <div class="catalog__container">
+
+        <div class="card" data-qa="card">
+          <div class="card__logo">
+            <img
+              src="./images/imac.jpeg"
+              alt="logo"
```

Comment:
It's a bad alt attribute, it should shortly describe what is pictured on this image